images/router/haproxy/conf/haproxy-config.template

```diff
@@ -100,15 +100,36 @@ frontend public
   acl secure_redirect base,map_beg(/var/lib/haproxy/conf/os_edge_http_redirect.map) -m found
   redirect scheme https if secure_redirect
 
+{{ if matchPattern "true|TRUE" (env "ROUTER_ALLOW_WILDCARD_ROUTES" "")}}
```

Comment:
Yeah, let's just support true for now.  The SYN eater I added got it backwards and added '1' so I fixed that later and support 'true' and '1', but 'true' is preferred.